salt/elife-xpub/scripts/setup-database.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
 set -e
 
-DC_COMMAND="docker-compose -f docker-compose.yml -f docker-compose.formula.yml"
-DB_CREATED_COMMAND="psql -c \"SELECT 'public.entities'::regclass\""
-DB_ENV="-e PGHOST=${PGHOST} -e PGPORT=${PGPORT} -e PGUSER=${PGUSER} -e PGDATABASE=${PGDATABASE} -e PGPASSWORD=${PGPASSWORD}"
-SETUP_ARGS="--username={{ pillar.elife_xpub.database.user }} --password={{ pillar.elife_xpub.database.password }} --email={{ pillar.elife_xpub.database.email }}"
+dc="docker-compose -f docker-compose.yml -f docker-compose.formula.yml"
+db_env="-e PGHOST=${PGHOST} -e PGPORT=${PGPORT} -e PGUSER=${PGUSER} -e PGDATABASE=${PGDATABASE} -e PGPASSWORD=${PGPASSWORD}"
+recreate="dropdb ${PGDATABASE} && createdb ${PGDATABASE}"
+
+# Removing setting up a user (via clobber) in PubSweet as at the moment this is no longer necessary
+# This may be re-instated after the work to upgrade to the latest PubSweet version.
+# Ref: elifesciences/elife-xpub/issues/1920
+#
+# SETUP_ARGS="--username={{ pillar.elife_xpub.database.user }} --password={{ pillar.elife_xpub.database.password }} --email={{ pillar.elife_xpub.database.email }}"
 
 if [ ! -z "${DROP}" ]; then
-    ${DC_COMMAND} run --rm app /bin/bash -c "npx pubsweet setupdb ${SETUP_ARGS} --clobber"
-    exit
+    ${dc} run --rm ${db_env} postgres /bin/bash -c "${recreate}"
 fi
 
-if ! ${DC_COMMAND} run --rm ${DB_ENV} postgres /bin/bash -c "${DB_CREATED_COMMAND}"
-then
-    ${DC_COMMAND} run --rm app /bin/bash -c "npx pubsweet setupdb ${SETUP_ARGS}"
-else
-    echo "Database is already present at ${PGHOST}:${PGPORT}"
-fi
+# Always run the migrate to ensure the database has the correct schema
```

Comment:
The command doesn't output much so worth turning this comment into an `echo ...`